.travis.yml

```diff
@@ -611,3 +612,5 @@ before_script:
   - gem install inspec rubocop
 script:
   - tox
+after_script:
```

Comment:
AFAICT, this will always run regardless of failure. See https://docs.travis-ci.com/user/job-lifecycle/.

Reply:
Yes.